tests/datadriven/test_variant_annotations.py

```diff
@@ -180,10 +180,10 @@ def _verifyVariantAnnotationsEqual(
                     transcriptEffects += self._splitCsqEffects(ann)
                 for treff, gaEffect in zip(
                         transcriptEffects,
-                        gaVariantAnnotation.transcriptEffects):
-                    self.assertEqual(gaEffect.alternateBases, treff['alt'])
-                    self.assertEqual(gaEffect.featureId, treff['featureId'])
-            self.assertIsNotNone(gaVariantAnnotation.transcriptEffects)
+                        gaVariantAnnotation.transcript_effects):
+                    self.assertEqual(gaEffect.alternate_bases, treff['alt'])
+                    self.assertEqual(gaEffect.feature_id, treff['featureId'])
+            # self.assertIsNotNone(gaVariantAnnotation.transcript_effects)
```

Comment:
Yes, this both can't be None and also gives an error "TypeError: Can only compare repeated composite fields against other repeated composite fields."
I'll replace this with an assertion over the length of this list.